src/api/app/helpers/webui/maintenance_incident_helper.rb

```diff
@@ -100,6 +56,43 @@ def release_targets_cell(incident)
 
   private
```

Comment:
AFAICS rails does not support private functions in helpers - the nature of mixing modules into controllers plainly makes them private functions to every controller.

So IMO you just pollute git blame without much of a gain here.

Reply:
> AFAICS rails does not support private functions in helpers - the nature of mixing modules into controllers plainly makes them private functions to every controller.

Not what I wanted to achieve. I will remove that part. Thanks for the hint.